crates/ruma-client-api/CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ Breaking changes:
   It will now return the proper status code and headers depending on the variant
   used.
 - The http crate had a major version bump to version 1.1
+- `RoomSummary::heroes` now properly contains only `UserId` instead of `String`
```

Comment:
If the string type existed only in a non-released git rev, I would personally say the changelog entry is better left out.

Reply:
(which is why generating changelogs from commit messages is stupid; thanks for coming to my TED talk)

Reply:
Oh this is code from 2022, so I recall there's been some releases since then :grin: But very nice TED talk, I feel very inspired now!